__arch_strncpy_from_user needs to be exported if you build the framebuffer console driver as a module.

Cheers,

Ian.

--- linux-2.6-bkpxa.orig/arch/arm/kernel/armksyms.c	2004-02-27 10:35:29.000000000 +0000
+++ linux-2.6-bkpxa/arch/arm/kernel/armksyms.c	2004-02-27 14:55:02.000000000 +0000
@@ -187,6 +187,7 @@
 EXPORT_SYMBOL(__arch_copy_to_user);
 EXPORT_SYMBOL(__arch_clear_user);
 EXPORT_SYMBOL(__arch_strnlen_user);
+EXPORT_SYMBOL(__arch_strncpy_from_user);
 
 	/* consistent area handling */
 EXPORT_SYMBOL(consistent_alloc);
